gnucash/report/reports/aging.scm

```diff
@@ -385,8 +385,8 @@ totals to report currency.")
        (N_ "Leading date.")
        'duedate
        (list
-         (vector 'duedate (N_ "Due Date") (N_ "Due date is leading.")) ;; Should be using standard label for due date?
-         (vector 'postdate (N_ "Post Date") (N_ "Post date is leading."))))) ;; Should be using standard label for post date?
+         (vector 'duedate (N_ "Due date is leading"))
```

Comment:
In this case I think it's superfluous to add the full description. The option name is already "Leading date". The "is leading" in the option values adds little IMO.

Reply:
When I first read this option, I was unsure of what it did, still don't, I assume it is some sort order but maybe I do not have enough dummy data to see. The option is 'Due or Post Date', the default tool tip was 'Leading Date'.

Reply:
Ok, I misinterpreted the option definition. The name of the option is "Due Date or Post Date" and the  tool tip is "Leading Date". That's an poor option name unfortunately, but I'm not to suggest option name changes in this PR.

Still your extended value names are not adding much compared to how the option  worked before. The "is leading" part would only appear in a tool tip anyway. So  before you would have a tool tip specifying "Due date is leading" or "Post date is leading". Now you have a common tip specifying "Leading date" and "Due date" or "Post date" as values. Adding the "is leading" part in the value names should not be necessary. If anything we could add a better explanation of what "Leading  date" means in the tool tip.